arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,410 @@
+                    "type": "string",
+                    "description": "Gets the storage account name of the Azure SQL Server auditing policy."
+                },
+                "storageAccountKey": {
+                    "type": "string",
+                    "description": "Gets the primary storage account key of the Azure SQL Server auditing policy."
+                },
+                "storageAccountSecondaryKey": {
+                    "type": "string",
+                    "description": "Gets secondary key of the storage account of the Azure SQL Server auditing policy."
+                },
+                "storageTableEndpoint": {
+                    "type": "string",
+                    "description": "Gets the storage table endpoint of the Azure SQL Server auditing policy."
+                },
+                "storageAccountResourceGroupName": {
+                    "type": "string",
+                    "description": "Gets the resource group of the storage account of the Azure SQL Server auditing policy."
+                },
+                "storageAccountSubscriptionId": {
+                    "type": "string",
+                    "description": "Gets the subscription Id of the storage account of the Azure SQL Server auditing policy."
+                },
+                "retentionDays": {
+                    "type": "string",
+                    "description": "Gets the retention in days of the Azure SQL Database auditing policy."
+                },
+                "auditLogsTableName": {
+                    "type": "string",
+                    "description": "Gets the name of the table audit logs are written to in the Azure SQL Database auditing policy."
+                },
+                "fullAuditLogsTableName": {
+                    "type": "string",
+                    "description": "Gets the full name of the table audit logs are written to in the Azure SQL Database auditing policy."
+                }
+            },
+            "description": "Represents the properties of an Azure SQL auditing policy."
+        },
+        "DatabaseAuditingPolicy": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/DatabaseAuditingPolicyProperties",
+                    "description": "Represents the properties of the resource."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "description": "Represents an Azure SQL Database auditing policy."
+        },
+        "ServerAuditingPolicyProperties": {
+            "properties": {
+                "auditingState": {
+                    "type": "string",
+                    "description": "Gets the auditing state of the Azure SQL Server auditing policy."
+                },
+                "eventTypesToAudit": {
```

Comment:
This is a list.

---
In reply to: [80346338](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80346338) [](ancestors = 80346338)